index.js

```diff
@@ -1,32 +1,43 @@
+const INITIAL_TIMER = 60 * 2 + 30;
 let interval,
   isPaused = false,
-  timer = 60 * 2 + 29,
+  timer = INITIAL_TIMER,
   minutes,
   seconds,
   isRunning = false,
-  display = document.querySelector("#time");
+  display = document.querySelector("#time"),
+  audio = document.querySelector("#audioBell");
 
 const images = ["./assets/Carlos.png", "./assets/Felipe.png"];
 
 window.onload = function () {
   document.getElementById("header-image").src =
     images[Math.round(Math.random() * 1)];
   hljs.initHighlightingOnLoad();
-};
 
-function startTimer(display) {
-  isRunning = true;
-  interval = setInterval(function () {
-    minutes = parseInt(timer / 60, 10);
-    seconds = parseInt(timer % 60, 10);
+  display.textContent = getMinSec(timer);
+
+};
 
-    minutes = minutes < 10 ? "0" + minutes : minutes;
-    seconds = seconds < 10 ? "0" + seconds : seconds;
+function getMinSec (timer) {
```

Comment:
I would change the name of this function, I don't know something like getReadableTime or seconds2Clock, seconds2idontknow. 

The idea is to let the developer that comes in the future to understand what this function does without having to look at the code. Maybe it could be as simple as:

This function transforms seconds to readable time, so I guess we could think of a name to indicate this.

Also, I would change the param to be called as what it means. I don't know what `timer` is, but in reality, this function transform `seconds` into something else. I would call that param `seconds` instead.